src/functions/indGraph.jl

```diff
@@ -0,0 +1,98 @@
+export IndGraph
+
+"""
+**Indicator of an operator graph**
+
+    IndGraph(A)
+
+For matrix `A` (dense or sparse) returns the indicator function of the set
+```math
+S = \\{(x, y) : Ax = y\\}.
+```
+
+The evaluation of `prox!` uses direct methods based on LDLt (LL for dense cases) matrix factorization and backsolve.
+
```

Comment:
I would here describe how the `Tuple`-friendly calls to function evaluation & `prox!` work: the whole library relies on the four-arguments `prox!`, the last argument `gamma` being optional. Therefore maybe it is better to describe the function in terms of single input/output objects. Something like

```
The `prox!` method operates on pairs `(x, y)` as input/output. So if `f = IndGraph(A)`,
while `(x, y)` and `(c, d)` are pairs of vectors of the same sizes, then `prox!((c, d), f, (x, y))`
writes to `(c, d)` the projection onto `S` of `(x, y)`.
```

Also, no need to spend words on the optional `gamma` below.